### ISSUE TITLE
Weird /² symbols in Riemann's Ideen. - Page numbers of an older source?

### ISSUE THREAD

**Issue by m-k94** [DCMLab/ddd#16]
![image](https://user-images.githubusercontent.com/75686579/104134846-7184fd80-538c-11eb-8c3e-bcfd737a9656.png)

I encounter this symbol about once every page in Riemann's text. On this particular page, it even breaks up a word. 

I don't think they are indicators of footnote or something, as for that purpose *) is used. My best guess is that it's designed to indicate page turns in the original publication? What do you think?

And second question is, what do we do with them? So far I am simply transcribing them as part of the text. If my hypothesis is correct, these numbers may be more interesting that the modern pagenumbers, as quotations from other sources are more likely to relate to them, I imagine.

**Comment by fabianmoss** [comment]
Yes these symbols indicate page breaks in the original source. You may notice that this is the only text that is not a scan of an original book. The reason is simply that I did not find one. Generally, this is a bad situation for editorial projects but not too bad for us. 

You are right that the modern page numbers are not interesting at all. It would be possible to mark them up as special page breaks using a label.

**Comment by m-k94** [comment]
Alright, thank you for confirming! 

For now, I would suggest that I keep continue copying the numbers as they appear. (We can easily find them all later rather easily because / is a rare symbol in the text). Maybe I can also just continue tagging the "new" page-numbers as such, so we can reinterpret them all later. Do you think this is a good way to proceed or should I change something now?

**Comment by fabianmoss** [comment]
I think the best would be to find a scan of the actual source than continuing with this rather weird edition. I will do another search but I believe I did that already several times so it might be we would actually have to order a scan by a library (which should be no problem). I'll think about that. For now, you can continue with the "modern" edition and markup the pages with some custom tag.
